cmd/subnetcmd/upgradecmd/generate.go

```diff
@@ -295,6 +301,37 @@ func promptRewardManagerParams(
 	return false, nil
 }
 
+func ConfigureInitialRewardConfig() (*rewardmanager.InitialRewardConfig, error) {
+	config := &rewardmanager.InitialRewardConfig{}
+
+	burnPrompt := "Should fees be burnt?"
```

Comment:
```suggestion
	burnPrompt := "By default, all transaction fees on Avalanche are burned (sent to a blackhole address). (Reward Manager Precompile)"
```

Answer options:
"I am fine with transaction fees being burned (Reward Manager Precompile OFF)"
"I want to customize the transaction fee distribution (Reward Manager Precompile ON)"